devsetup/scripts/standalone.sh

```diff
@@ -53,6 +53,7 @@ MANILA_ENABLED=${MANILA_ENABLED:-true}
 SWIFT_REPLICATED=${SWIFT_REPLICATED:-false}
 TLSE_ENABLED=${TLSE_ENABLED:-false}
 CLOUD_DOMAIN=${CLOUD_DOMAIN:-localdomain}
+OCTAVIA_ENABLED=${OCTAVIA_ENABLED:-true}
```

Comment:
where does this variable get used?

Reply:
Good question. Where does `BARBICAN_ENABLED` and `MANILA_ENABLED` get used? Nowhere. That line should have no effect. I'll remove it.